app/src/JokeFetch.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import BackButton from "./BackButton";
+import NextButton from "./NextButton";
+
+const buttonText = [
+  "Ugh.",
+  "🤦🏻‍♂️",
+  "omg dad.",
+  "you are the worst",
+  "seriously",
+  "stop it.",
+  "please stop",
+  "that was the worst one",
+];
+
+const JokeFetch = () => {
+  /* Render json from fetchJoke()
+     when handleClick() event fires */
+  const [joke, setJoke] = useState("");
+  /* Toggle button with boolean values  */
+  const [toggle, setToggle] = useState(false);
+  /* Randomise array string value "Joke Review"
+     onClick button event fires */
+  const [index, setIndex] = useState(-1);
+
+  const startName = [""];
+  const randomJokeReview = index !== -1 ? buttonText[index] : startName[0];
+
+  async function fetchJoke() {
```

Comment:
Maybe consider using some error handling, it is super helpful to use throw new error incase the data cannot be found